### DISCUSSION TITLE
RTL props / support

### DISCUSSION THREAD

**Discussion by etabackman** [mantinedev/mantine#492] [Q&A] [answered]
I am evaluating Mantine for use in a production app that will require support for both LTR and RTL, depending on the user's preference.

In reviewing the documentation (it looks **amazing**, but I have not yet actually used the library), I see that some components have names or props like `SelectRightSection`,  `rightSection`, `rightIcon`, `leftIcon`...

In RTL mode, do these automatically move to the other side?

Is ongoing RTL support high priority for this library?

Are there any known RTL gotchas?

Thank you!

**Comment by rtivital** [comment, marked as answer]
Yes, all above props will change direction as expected. There are several known issues with RTL, which we will be fixing in one of the future releases (see screenshot). Other components that are not listed below were tested to work well with RTL.

<img width="252" alt="Снимок экрана 2021-12-07 в 15 53 27" src="https://user-images.githubusercontent.com/10353856/145032545-65d45858-ce69-4d72-a567-2a72aefdbecb.png">

**Reply by etabackman** [reply]
7 minute response time! You are incredible!

Thanks. Very excited about this library.
